### ISSUE TITLE
improve the navbar

### ISSUE THREAD

**Issue by kirtiraj22** [kirtiraj22/Frontend-Bootcamp-Website#1]
Make some improvements in navbar to make it a bit attractive.

**Comment by srijoy-paul** [comment]
@kirtiraj22 Can you assign me this task?

**Comment by ElectricPlant19** [comment]
assign this to me

**Comment by kirtiraj22** [comment]
assigning it to both!
it will be first come first serve!

and make sure to star this repo and follow me!

**Comment by srijoy-paul** [comment]
I think it should be assigned to one who commented first and if in 24 hours it's not completed, then pass to next
In this way both will work on the same thing @kirtiraj22

**Comment by ElectricPlant19** [comment]
> I think it should be assigned to one who commented first and if in 24 hours it's not completed, then pass to next In this way both will work on the same thing @kirtiraj22

fair enough

**Comment by kirtiraj22** [comment]
alright as both of you are convinced it will be assigned to @srijoy-paul 
you can start working on it @srijoy-paul 

and also @ElectricPlant19 if you want to work on this website then you can find any other issue in the website then create a new issue and I will assign it to you!

**Comment by ElectricPlant19** [comment]
i have added a new issue can you check.

**Comment by kirtiraj22** [comment]
> i have added a new issue can you check.

done!

**Comment by ElectricPlant19** [comment]
its been more than 24 hours if the work is still not done then i guess i should be assigned

**Comment by kirtiraj22** [comment]
@ElectricPlant19 yaa go for it!!

**Comment by Hicode-dev** [comment]
can i work on the navbar

**Comment by kirtiraj22** [comment]
@Hicode-dev sure mate!

I hope you are comfortable with bootstrap

**Comment by Hicode-dev** [comment]
Yes sure I am .

**Comment by Hicode-dev** [comment]
So how I do I get started should I create a branch first?

**Comment by Hicode-dev** [comment]
Please assign me back 🙏

**Comment by Hicode-dev** [comment]
> @Hicode-dev sure mate!
> 
> I hope you are comfortable with bootstrap

am actually done with it buh you need to give me permission to pushh

**Comment by Hicode-dev** [comment]
am actually done with it buh you need to give me permission to pushh

**Comment by kirtiraj22** [comment]
@Hicode-dev, assigned it back to you.
you can make a PR now

**Comment by Hicode-dev** [comment]
yes but what branch am i pushing into and also you have not me a contributor

**Comment by kirtiraj22** [comment]
@Hicode-dev bro you can make a separate branch if you want or else you can push the code to the main branch directly(as this is not a big project), and the issue is already assigned to you!

**Comment by Hicode-dev** [comment]
yes i know i tried that but its telling me that i need a permission from you to push and i can create a branch unless and added as a contributor

**Comment by Hicode-dev** [comment]
i tried a PR buh it wont work and thats beacuse youve not made me a contributor

**Comment by kirtiraj22** [comment]
can you share the screenshot of the error?

**Comment by kirtiraj22** [comment]
hey @Hicode-dev you can't directly push to the owner's repo. 
- you need to fork this repo first 
- then clone it to you local machine 
- make a branch (optional)
- commit the changes (to the forked repo)
- and then create a PR to my repo
